app/views/planning_applications/_planning_application_table.html.erb

```diff
@@ -3,18 +3,8 @@
   <table class="govuk-table">
     <thead class="govuk-table__head">
     <tr class="govuk-table__row">
-      <th scope="col" class="govuk-table__header">Application number</th>
-      <th scope="col" class="govuk-table__header">Site address</th>
-      <th scope="col" class="govuk-table__header">Application type</th>
-      <th scope="col" class="govuk-table__header">Expiry date</th>
-      <% if id != "closed" %>
-        <th scope="col" class="govuk-table__header">Days left to expiry</th>
-      <% end %>
-      <th scope="col" class="govuk-table__header">Status</th>
-      <% if id == "awaiting_determination" %>
-        <th scope="col" class="govuk-table__header">Recommendation date</th>
-      <% elsif id == "closed" %>
-        <th scope="col" class="govuk-table__header">Determination date</th>
+      <% header_options(id).each do |option| %>
```

Comment:
I know this existed already in the code but it's a bit confusing to have "id" mean the "status" here imo. Should we update the rendering of these partials to reflect what this really is.

Reply:
you're right, id is confusing - I've changed it to "planning_application_status" so its more readable